src/xdocs/config_naming.xml

```diff
@@ -658,6 +658,41 @@ class MyClass3&lt;abc&gt; {} // violation, the class type parameter
         <source>
 &lt;module name="ConstantName"/&gt;
         </source>
```

Comment:
Please add code example for default config

Reply:
Done